lib/Doctrine/DBAL/DriverManager.php

```diff
@@ -90,17 +90,17 @@ private function __construct()
      *
      * Either 'driver' with one of the following values:
      *
+     *     drizzle_pdo_mysql
```

Comment:
Hm… it actually duplicates the keys of the driver map. Would it make sense to reference if using `{@link $_driverMap}` instead of duplication?

https://github.com/doctrine/dbal/blob/3d6d3542a32d0af28136db9d3a06614b32298a48/lib/Doctrine/DBAL/DriverManager.php#L46-L58

Reply:
True, I am fine with whatever. Although `$_driverMap` doesn't seem to be listed in the [API docs](https://www.doctrine-project.org/api/dbal/latest/Doctrine/DBAL/DriverManager.html), which I would consider to be quite useful.

Reply:
One more note, then I'll be happy to push whatever you find to be the best solution:
a) Which one of these would you prefer? Using a `{@link $_driverMap}` or just linking to the `getAvailableDrivers()` function which returns the keys of this array.
b) The docblock has no return value annotation so if I'm here, I might as well fix that too. Should I?

Also, what is your commit squashing policy? Three commits feel a bit much for a one line documentation change.

Reply:
>Which one of these would you prefer?

`{@link $_driverMap}` seems like a more reasonable reference. It contains the actual values we're referring to.

> The docblock has no return value annotation […]

Which one?

> Also, what is your commit squashing policy? Three commits feel a bit much for a one line documentation change.

We usually squash commits unless they provide additional clarity to the changes (not the case here).

Reply:
Oh, and `getConnection()` is missing the return statement.

Reply:
> Oh, and `getConnection()` is missing the return statement.

As for the parameter and return types, there's some work in progress in https://github.com/doctrine/dbal/pull/3799. I'd rather address such issues in a systems way rather than one by one.